library/core/src/clone.rs

```diff
@@ -7,11 +7,7 @@
 //! contain owned boxes or implement [`Drop`]), so the compiler considers
 //! them cheap and safe to copy. For other types copies must be made
 //! explicitly, by convention implementing the [`Clone`] trait and calling
-//! the [`clone`][clone] method.
-//!
-//! [`Clone`]: trait.Clone.html
-//! [clone]: trait.Clone.html#tymethod.clone
-//! [`Drop`]: ../../std/ops/trait.Drop.html
+//! the [`Clone::clone`] method.
```

Comment:
Let's keep the text that was here before, it's less repetitive.
```suggestion
//! the [`clone`] method.
//!
//! [`clone`]: Clone::clone
```

Reply:
I was thinking about the same and chose the `Clone::clone` one, but you're right it looks clearer.